<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Uart</class>
 <widget class="QWidget" name="Uart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 52, 54);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(186, 189, 182);</string>
           </property>
           <property name="text">
            <string>串    口: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="serials_box">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(238, 238, 236);</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(186, 189, 182);</string>
           </property>
           <property name="text">
            <string>波特率:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="baudrate">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(238, 238, 236);</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="openPortBtn">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(238, 238, 236);</string>
         </property>
         <property name="text">
          <string>打开端口</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closePortBtn">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(238, 238, 236);</string>
         </property>
         <property name="text">
          <string>关闭端口</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sendBtn">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(211, 215, 207);</string>
         </property>
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="resvBtn">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(211, 215, 207);</string>
         </property>
         <property name="text">
          <string>接收</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="dataText">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>250</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>250</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeBtn">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(211, 215, 207);</string>
       </property>
       <property name="text">
        <string>关  闭</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
